pkg/ensurance/collector/nodelocal/memory.go

```diff
@@ -0,0 +1,61 @@
+package nodelocal
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/shirou/gopsutil/mem"
+	"k8s.io/klog/v2"
+
+	"github.com/gocrane/crane/pkg/common"
+	"github.com/gocrane/crane/pkg/ensurance/collector/types"
+)
+
+const (
+	memoryCollectorName = "memory"
+)
+
+func init() {
+	registerMetrics(memoryCollectorName, []types.MetricName{types.MetricNameMemoryTotalUsage, types.MetricNameMemoryTotalUtilization}, NewMemoryCollector)
+}
+
+type MemoryCollector struct {
+	data map[string][]common.TimeSeries
+}
+
+// NewMemoryCollector returns a new Collector exposing kernel/system statistics.
+func NewMemoryCollector(_ *NodeLocalContext) (nodeLocalCollector, error) {
+
+	klog.V(2).Infof("NewMemoryCollector")
```

Comment:
such log is can be removed, no use for real deployment.

Reply:
done~

Reply:
done~